eitprocessing/mixins/slicing.py

```diff
@@ -119,11 +113,15 @@ def __getitem__(self, key: slice | int | float):
         if isinstance(key, slice):
             if key.step:
                 raise ValueError("Can't slice by time using specific step sizes.")
-            start_value = key.start or self.obj.time[0]
-            end_value = key.stop or np.inf
+            if start_value is None:
+                start_value = self.obj.time[0]
+            if end_value is None:
+                end_value = np.inf
```

Comment:
`start_value` and `end_value` can just be None. It is not necessary that they
have values.